tests/installation/zdup_prepare_repos.pm

```diff
@@ -0,0 +1,99 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: prepare repos for zdup.test
+# Maintainer: Oliver Kurz <okurz@suse.de>
+
+use base "installbasetest";
+use strict;
+use warnings;
+use testapi;
+use utils 'OPENQA_FTP_URL';
+
+sub run {
+    my $self = shift;
+
+    # precompile regexes
+    my $zypper_packagekit       = qr/^Tell PackageKit to quit\?/m;
+    my $zypper_packagekit_again = qr/^Try again\?/m;
+    my $zypper_repo_disabled    = qr/^Repository '[^']+' has been successfully disabled./m;
+
+    # This is just for reference to know how the network was configured prior to the update
+    script_run "ip addr show";
+    save_screenshot;
+
+    # before disable we need to have cdrkit installed to get proper iso appid
+    script_run "zypper -n in cdrkit-cdrtools-compat";
```

Comment:
In https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7633 I have removed this call. Please be aware of this when rebasing

Reply:
Thanks.